design/backend/templates/views/payments/components/cc_processors/razorpay/razorpay.tpl

```diff
@@ -23,8 +22,8 @@
 <div class="form-field">
     <label  for="currency">{__("currency")}:</label>
         <select name="payment_data[processor_params][currency]" id="currency">
-            {foreach from=$razorpay_currencies key="key" item="currency"}
-                <option value="{$key}" {if !isset($currencies.$key)} disabled="disabled"{/if} {if $processor_params.currency == $key} selected="selected"{/if}>{__({$currency})}{$currencies.$key}</option>
+            {foreach $currencies as $code => $currency}
```

Comment:
@sonyy-m , don't see it's like to like change. How to display other currency supported by RZP.